Structure_Engine/Structure_Engine.csproj

```diff
@@ -49,6 +49,10 @@
       <HintPath>..\..\BHoM\Build\Data_oM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Dimensional_oM, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Reference should not contain version, culture, or processorArchitecture.
Reference should also have `copy local` set to false.